### ISSUE TITLE
The "Controls" button is not intuitive because it looks like a "full screen" action

### ISSUE THREAD

**Issue by nomis** [liorhass/UsbTerminal#5]
The "Controls" icon:
![Four identical arrows in a square pointing horizontally/vertically from the centre to the four sides up/down/left/right](https://github.com/liorhass/UsbTerminal/assets/70171/a8e8521e-c0e4-445c-b72f-0d994939a1dd)

Is easily confused for a "full screen" icon:
![Four identical arrows in a square pointing diagonally from the centre to the four corners](https://github.com/liorhass/UsbTerminal/assets/70171/f80b9f41-8fe7-4fd0-999c-0e665314f99a)

**Comment by liorhass** [comment]
I agree. This button toggles the "arrow buttons" line (so the user can use the arrow buttons).
Any idea for a better icon? (preferably from the Android build-in icon set)

**Comment by nomis** [comment]
These might work together, but it relies on the `< >` "code" being interpreted as "lower level/special functions" or perhaps "left/right arrows":
![ic_baseline_code](https://github.com/liorhass/UsbTerminal/assets/70171/5d1ed45c-80fc-4510-937e-a0087e010167) https://icon-sets.iconify.design/ic/baseline-code/
![ic_baseline_code_off](https://github.com/liorhass/UsbTerminal/assets/70171/bf2b93c4-bac8-467d-aabf-962bd65ec35a) https://icon-sets.iconify.design/ic/baseline-code-off/

These could be used to indicate "something else you can do" without looking like "full screen":
![ic_baseline_widgets](https://github.com/liorhass/UsbTerminal/assets/70171/ce7361c7-39cb-46a4-b068-dec25fe91b64) https://icon-sets.iconify.design/ic/baseline-widgets/
![ic_baseline_tune](https://github.com/liorhass/UsbTerminal/assets/70171/c2904588-7b10-4202-8654-1c6c1ddf3c0a) https://icon-sets.iconify.design/ic/baseline-tune/

This looks like the menu bar that appears but that's just a co-incidence so it may not stay like that forever:
![ic_baseline_call_to_action](https://github.com/liorhass/UsbTerminal/assets/70171/9763d5f0-6ecd-4f5b-85f5-8207f59ef37e) https://icon-sets.iconify.design/ic/baseline-call-to-action/

The tab icon would also work but it could be interpreted as only a way to insert a tab character.
